### PR TITLE
Adding owners for label

### PR DESCRIPTION
**Pull request by aritraghosh**
Auto assign owners when labelled - Previously, owners were assigned only when the issue is opened with the label. Wit this change, the owners will get assigned when they get labelled (even when opened)
Removed the test label change - This was for testing some of the GH changes as there is no way to test this in private repos

### CODE REVIEW

**Review by julia-yin — APPROVED** [review]

**Comment by AllenWen-at-Azure** [comment]
There are still rules with "**assignMentionees: False**", is that because the corresponding assignees are out of AKS team?

File: .github/policies/resourceManagement.yml [hunk 1]
```diff
@@ -349,18 +349,28 @@ configuration:
       - or:
         - payloadType: Issues
       - isAction:
-          action: Opened
+          action: Labeled
```
Comment (Tatsinnit) [inline .github/policies/resourceManagement.yml:352]: 💡Observation share only: Worth to add context for this change of table please.

Comment (aritraghosh) [reply .github/policies/resourceManagement.yml:352]: Added

**Review by Tatsinnit — APPROVED** [review]
👍☕️ looks good to me. There is a merge conflict if you can please look into. Happy to merge post that is no one else has any feedback. I see this PR is open for a very long time. Thanks heaps.

**Pull request merged**